students/sbolsen/lesson03/strformat_lab.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+#TASK_ONE
+a = (2, 123.4567, 10000, 12345.67)
+print("file_{:03d}: {:03.2f} {:0.2E} {:0.2E}".format(a[0], a[1], a[2], a[3]))
+
+#TASK_TWO
+
+a = (2, 123.4567, 10000, 12345.67)
+print('file_{first}: {second} {third} {fourth}'.format(first = "{:03d}".format(a[0]), second = "{:03.2f}".format(a[1]), third = "{:0.2E}".format(a[2]), fourth = "{:0.2E}".format(a[3])))
+
+#TASK_THREE
+
+def formatter(tuple_in):
+    x = tuple_in
+    y = len(tuple_in)
```

Comment:
It's a good idea in general to use meaningful variable names, and in this case, it might be more explicit to just pass `len(tuple_in)` directly into `format()` without reassigning